wiki/Tournaments/GTS/RGTS_2022/en.md

```diff
@@ -0,0 +1,187 @@
+   - In case the players cannot come to an agreement, they will put in 2 teams with at least 3 members in each in order of global ranking.
+   - If the players cannot agree or find a captain, the highest ranked player will be chosen.
+8. The captain has the responsibility to reschedule a match if needed and will ensure their teammates are present for the matches.
+
+### Stage regulations
+
+1. There are seven stages in the tournament: the Qualifiers, the Group stage, the Round of 16, the Quarterfinals, the Semifinals, the Finals, and the Grand Finals.
+2. Teams will be seeded with the average of their ranks on each map during the Qualifiers round in 4 seeds: Top seed, High seed, Low seed, and Unseeded.
+3. Teams will be drawn in **8 groups of 4 teams** during live drawings which will happen on Sunday, 6 November 2022 (18:00 UTC).
+4. In the Group stage, all the teams from each group will face each other. The **best 2 teams** advance from their groups.
+5. Rankings of each group are determined by sorting the results of each team's performance in the following priority:
+   1. Most matches won.
+   2. Highest "score" (number of beatmaps won - number of maps lost).
+   3. Most beatmaps won.
+   4. Winner of the match played previously between the tied teams.
+6. During the Group stage, winning 5:0 or having +1.0 score difference ratio will be considered a win by default.
+7. Each stage has the following win conditions:
+   - **Group Stage**: Best-of-9 (first to 5 points)
+   - **Round of 16, and the Quarterfinals**: Best-of-11 (first to 6 points)
+   - **Semifinals, the Finals, and the Grand Finals**: Best-of-13 (first to 7 points)
+
+### Qualifier regulations
+
+1. During this round there will be multiple lobbies available at various hours.
+2. A private match will be created and teams will be invited into the lobby when it is their time to play. Make sure to be online at this time.
+3. There will be no warmups during this stage.
+4. Failed scores will count.
+5. Only two players from each team will be playing per map. There are no limitations on player swapping.
+6. Seeding will be done with the average rank of each team on each map.
+7. Maps will be played twice in a row during the lobby. The best run will be taken into account for the ranking.
+8. Only the 32 best teams will continue on to the bracket stage.
+
+### Mappool instructions
+
+1. There will be one mappool for each stage.
+2. The loser's bracket will play on the same pool as the winner's bracket of the same weekend.
+3. The Qualifier pool will be different from all other rounds, as it will have a format of 3 NoMod maps, 2 Hidden map, 2 HardRock map, 1 DoubleTime map, and 1 FreeMod map.
```

Comment:
I forgot, has it been decided to spell mods as usually in tournament articles or not (`Double Time` and such)?

Reply:
I'm pretty sure it hasn't, still open and is #8121 if I can recall correctly. So let's keep it for now, before there'll be a script to replace (maybe) just like how new flag implementation was also held.

Reply:
will be replaced by script at some point when i get to it (https://github.com/ppy/osu-wiki/issues/7401), so it doesn't matter rn. owc uses spaced for instance